shared/chat/inbox/index.desktop.js

```diff
@@ -115,38 +183,51 @@ class Inbox extends PureComponent<void, Props, void> {
           onHotkey={this.props.onNewChat}
         />
         {this.props.showNewConversation && <NewConversation />}
-        <div style={scrollableStyle} onScroll={this._onScroll}>
+        <div style={_scrollableStyle} onScroll={this._onScroll}>
           <ReactList
             ref={this._setRef}
-            style={listStyle}
             useTranslate3d={true}
             itemRenderer={this._itemRenderer}
             length={this.props.rows.count()}
             type="variable"
             itemSizeGetter={this._itemSizeGetter}
           />
         </div>
+        {this.state.showFloating &&
+          this.props.showSmallTeamsExpandDivider &&
+          <FloatingDivider
+            toggle={this.props.toggleSmallTeamsExpanded}
+            badgeCount={this.props.bigTeamsBadgeCount}
+          />}
+        {/*
```

Comment:
this is in the design but it should lead to the teams tab which doesn't exist yet